lib/handlers/event/push/rendering/PushNodeRenderers.ts

```diff
@@ -505,7 +505,7 @@ export class K8PodNodeRenderer extends AbstractIdentifiableContribution
                     envs.push(env);
                 }
                 pod.containers.forEach(c => {
-                    if (c.state === "running") {
+                    if (c.state === "running" && c.ready === true) {
```

Comment:
I guess there is no "ready" state?

Reply:
Seems not. Something can be running but not ready - that's a k8s thing.

Reply:
Right, but not an Atomist thing. Seems reasonable if we are only reporting running then we should add this check to combine Kubernetes ideas of ready and running.